### DISCUSSION TITLE
How i do pyston run in tox

### DISCUSSION THREAD

**Discussion by ozkansen** [tox-dev/tox#2292] [Q&A] [answered]
Hi,
How i do pyston run in tox.

pyston: https://github.com/pyston/pyston

Thanks

**Comment by undingen** [comment, marked as answer]
I'm not familiar at all with tox but I tried starting to add support for Pyston:
https://github.com/undingen/tox/commit/fac2ced92023101c29af6ff1a96e9407b8a286af
This is very WIP and I'm currently on holiday and not working on it but after reading your Issue I thought I will upload it because it may help.

Also here is the issue on the Pyston tracker: https://github.com/pyston/pyston/issues/151
